build-info-api/src/main/java/org/jfrog/build/api/Vcs.java

```diff
@@ -41,6 +42,31 @@ public String toString() {
                 ", url='" + url + '\'';
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof Vcs)) {
+            return false;
+        }
+
+        Vcs that = (Vcs) o;
+
+        if (!Objects.equals(revision, that.revision)) {
```

Comment:
Can be simplified to:
return (Objects.equals(revision, that.revision) && Objects.equals(url, that.url));